### PR TITLE
Allow WA Updates from chat links.

### PR DESCRIPTION
**Pull request by Grimonja**
# Description
 This PR allows WA Update from a chat link. All functionality was already in WA just not used.

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: WeakAuras/Transmission.lua [hunk 1]
```diff
@@ -1897,13 +1897,21 @@ Comm:RegisterComm("WeakAuras", function(prefix, message, distribution, sender)
     if(received.m == "d") and safeSenders[sender] then
       tooltipLoading = nil;
       local data, children, icon, icons = received.d, received.c, received.i, received.a
+      local target
+      for _, installedData in pairs(WeakAurasSaved.displays) do
```
Comment (emptyrivers) [inline WeakAuras/Transmission.lua:1901]: MatchInfo tries to do this for you, did that not work?

File: WeakAurasOptions/WeakAurasOptions.lua [hunk 2]
```diff
@@ -3782,7 +3784,7 @@ function WeakAuras.PositionOptions(id, data, _, hideWidthHeight, disableSelfPoin
       name = L["Anchored To"],
       order = 72,
       hidden = IsParentDynamicGroup,
-      values = (data.regionType == "group" or data.regionType == "dynamicgroup") and WeakAuras.anchor_frame_types_group or WeakAuras.anchor_frame_types,
+      values = WeakAuras.anchor_frame_types,
```
Comment (emptyrivers) [inline WeakAurasOptions/WeakAurasOptions.lua:3787]: ???

File: WeakAurasOptions/WeakAurasOptions.lua [hunk 3]
```diff
@@ -3821,7 +3823,7 @@ function WeakAuras.PositionOptions(id, data, _, hideWidthHeight, disableSelfPoin
           return L["To Screen's"]
         elseif (data.anchorFrameType == "PRD") then
           return L["To Personal Ressource Display's"];
-        else
+        elseif (data.anchorFrameType == "SELECTFRAME" or data.anchorFrameType == "CUSTOM") then
```
Comment (emptyrivers) [inline WeakAurasOptions/WeakAurasOptions.lua:3826]: ???

**Comment by emptyrivers** [comment]
image
You've got a build error, too

File: WeakAuras/Transmission.lua (hunk 1 above)
Comment (Grimonja) [reply WeakAuras/Transmission.lua:1901]: It appears it does. Ill remove that unecesary code.

File: WeakAurasOptions/WeakAurasOptions.lua [hunk 4]
```diff
@@ -1295,8 +1295,11 @@ function WeakAuras.DeleteOption(data, massDelete)
 
   frame:ClearPicks();
   WeakAuras.Delete(data);
-  frame.buttonsScroll:DeleteChild(displayButtons[id]);
-  displayButtons[id] = nil;
+
+  if(displayButtons[id])then
```
Comment (InfusOnWoW) [inline WeakAurasOptions/WeakAurasOptions.lua:1299]: I don't understand how this can happen.

Comment (Grimonja) [reply WeakAurasOptions/WeakAurasOptions.lua:1299]: It happens when you import a update WA if you never opened WA options before. As buttons are created in "async" way.

**Pull request merged**